### PR TITLE
Fix Codacy Issues

### PR DESCRIPTION
**Pull request by domdomegg**

### CODE REVIEW

File: upload-to-commons.patch [hunk 1]
```diff
@@ -1,21 +0,0 @@
-diff --git a/app/src/main/java/fr/free/nrw/commons/CommonsApplication.java b/app/src/main/java/fr/free/nrw/commons/CommonsApplication.java
```
Comment (whym) [inline upload-to-commons.patch]: What is the reason for deleting this file?

Comment (domdomegg) [inline upload-to-commons.patch]: Unused file - No longer required as has already been applied

Comment (whym) [inline upload-to-commons.patch]: Oh I see. In that case, I would mention that explicitly in the commit message (or commit title). "Fix Codacy Issues" sounds a bit indirect.

**Comment by domdomegg** [comment]
Renamed commit

**Pull request merged**